hail_search/queries/mito.py

```diff
@@ -118,6 +121,156 @@ class MitoHailTableQuery(BaseHailTableQuery):
         CLINVAR_KEY: 'clinvar_path_variants.ht',
     }
 
+    def _import_and_filter_entries_ht(
+        self, project_guid: str, num_families: int, project_sample_type_data, **kwargs
+    ) -> tuple[hl.Table, hl.Table]:
+        sample_types = set(project_sample_type_data.keys())
+        if len(sample_types) == 1:
+            # Import and filter entry table for one family or one project with one sample type
+            return super()._import_and_filter_entries_ht(project_guid, num_families, project_sample_type_data, **kwargs)
+
+        self._has_both_sample_types = True
+        entries = {}
+        for sample_type in sample_types:
+            ht, sample_data = self._load_family_or_project_ht(
+                num_families, project_guid, project_sample_type_data, sample_type, **kwargs
+            )
+            entries[sample_type] = (ht, sample_data)
```

Comment:
nitpciky, but slighlty neater to do this in one line: 
```suggestion
            entries[sample_type] = self._load_family_or_project_ht(
                num_families, project_guid, project_sample_type_data, sample_type, **kwargs
            )
```